packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -122,11 +123,21 @@ export class HostedPluginProcess implements ServerPluginRunner {
         const hostedPluginManager = rpc.getProxy(MAIN_RPC_CONTEXT.HOSTED_PLUGIN_MANAGER_EXT);
         hostedPluginManager.$stopPlugin('').then(() => {
             emitter.dispose();
-            cp.kill();
+            this.killProcessTree(cp);
         });
 
     }
 
+    private killProcessTree(parentProcess: cp.ChildProcess): void {
+        const parentPid = parentProcess.pid;
+        // tslint:disable-next-line:no-any
+        psTree(parentPid, (err: Error, children: Array<any>) => {
+            parentProcess.kill();
+            // tslint:disable-next-line:no-any
+            cp.spawn('kill', children.map((p: any) => p.PID));
```

Comment:
Will it work on windows? Would it make sense to run `taskkill` on Windows instead like here: https://github.com/microsoft/vscode/blob/a848f18df4efa9eb947bd01cbebcede418e66148/src/vs/workbench/contrib/debug/node/debugAdapter.ts#L269-L275

Reply:
I have tested the new commit on windows 10.
But then I found that when I close the browser page, there many errors which may related with some message received by plugin-host process , so I disconnect the parentProcess at first.
```
UnhandledPromiseRejectionWarning: Error: Unknown actor WorkspaceMain
...
UnhandledPromiseRejectionWarning: Error: Unknown actor LanguagesMain
...
UnhandledPromiseRejectionWarning: Error: Unknown actor CommandRegistryMain
...
```

Reply:
@akosyakov Please let me know if you have any questions. :)